src/Console/Database/CheckSchemaIntegrityCommand.php

```diff
@@ -129,6 +130,13 @@ protected function configure()
             InputOption::VALUE_NONE,
             __('Check tokens related to migration from signed to unsigned integers in primary/foreign keys.')
         );
+
+        $this->addOption(
+            'plugin',
+            'P',
```

Comment:
We usually use lowercase.
```suggestion
            'p',
```